test/network/2020-03-01/armnetwork/zz_generated_constants.go

```diff
@@ -1201,15 +1201,15 @@ func (c ExpressRouteLinkConnectorType) ToPtr() *ExpressRouteLinkConnectorType {
 type ExpressRouteLinkMacSecCipher string
 
 const (
-	ExpressRouteLinkMacSecCipherGCMAES128 ExpressRouteLinkMacSecCipher = "gcm-aes-128"
-	ExpressRouteLinkMacSecCipherGCMAES256 ExpressRouteLinkMacSecCipher = "gcm-aes-256"
+	ExpressRouteLinkMacSecCipherGCMAes128 ExpressRouteLinkMacSecCipher = "gcm-aes-128"
```

Comment:
I think something is wrong here...AES should be in all caps too

Reply:
Fixed